src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -189,13 +206,33 @@ protected Config parseConfiguration(final String[] args) {
             }
         }
 
+        validateConfig(config);
+
         // Write command line options to disk
         saveConfig(config);
 
         return config;
     }
 
     /**
+     * Validation routines for checking interdependent configuration parameters.
+     * @param config
+     */
+    private void validateConfig(final Config config) {
+        // verify bagit config
+        if (config.getBagProfile() != null &&
+                config.getBagConfigPath() == null) {
+            throw new RuntimeException("A bagit config path must be set when using a bagit profile.");
```

Comment:
In theory, you could have a profile that didn't require any user-supplied metadata.  Even our default profile is requiring the organization name, but you could have a profile called "minimal" that would export valid bags with no extra metadata at all.  So having the option to skip this if there were no required fields seems reasonable.